teos/cli/teos_cli.py

```diff
@@ -54,36 +83,66 @@ def main(command, args, command_line_conf):
 
     setup_data_folder(DATA_DIR)
 
+    teos_rpc_host = config.get("RPC_BIND")
+    teos_rpc_port = config.get("RPC_PORT")
+
+    rpc_client = RPCClient(teos_rpc_host, teos_rpc_port)
+
     try:
         if command == "get_all_appointments":
-            appointment_data = get_all_appointments(config.get("RPC_BIND"), config.get("RPC_PORT"))
-            if appointment_data:
-                print(json.dumps(appointment_data, indent=4, sort_keys=True))
+            result = rpc_client.get_all_appointments()
+
+        elif command == "get_tower_info":
+            result = rpc_client.get_tower_info()
+
+        elif command == "get_users":
+            result = rpc_client.get_users()
+
+        elif command == "get_user":
+            if not args:
+                sys.exit("No user_id was given")
+            if len(args) > 1:
+                sys.exit(f"Expected only one argument, not {len(args)}")
 
+            result = rpc_client.get_user(args[0])
         elif command == "help":
             if args:
                 command = args.pop(0)
 
                 if command == "get_all_appointments":
                     sys.exit(help_get_all_appointments())
 
+                elif command == "get_tower_info":
+                    sys.exit(help_get_tower_info())
+
+                elif command == "get_users":
+                    sys.exit(help_get_users())
+
+                elif command == "get_user":
+                    sys.exit(help_get_user())
+
                 else:
                     sys.exit("Unknown command. Use help to check the list of available commands")
 
             else:
                 sys.exit(show_usage())
 
+    except grpc.RpcError as e:
```

Comment:
This may need better handling to provide a proper feedback to the user. Split the error based on the error code and provide some good reasons to the errors.

For example, if the tower is not online you'll get `failed to connect to all addresses` atm, which is not descriptive at all.

Reply:
I added a tailored message for status code UNAVAILABLE, but decided against further splitting by status code, as in all the other cases that I know of the `details()` call should provide an appropriate description, as we generate them in the internal api. 8a8a2aae2ef3f1ddc05d732f9f9f96764447d0eb